shared/search/user-pane/index.js

```diff
@@ -30,10 +30,7 @@ export default connector.connect(
       if (username && trackerState && trackerState.type === 'tracker') {
         const {userInfo: {uid}} = trackerState
         const currentlyFollowing = trackerState.lastAction === 'followed' || trackerState.lastAction === 'refollowed' || trackerState.currentlyFollowing
-        // TODO (mm) ideally userInfo should be null until we get a response from the server
-        // Same with proofs (instead of empty array). So we know the difference between
-        // not having data and having empty data.
-        const loading = trackerState.userInfo.followersCount === -1
+        const loading = trackerState.serverActive
```

Comment:
lets use the serverActive flag. this is true between start/finish. we can get a start/finish w/o loading the usercard so the followers count isn't a good indicator of the loading state